leapp/utils/output.py

```diff
@@ -1,10 +1,17 @@
 from __future__ import print_function
+
 import json
+import os
 import sys
 from contextlib import contextmanager
 
-from leapp.models import ErrorModel
 from leapp.exceptions import LeappRuntimeError
+from leapp.models import ErrorModel
+
+
+def _is_verbose():
+    """Redefinition of is_verbose from leapp.libraries.stdlib.config because it leads to import errors"""
```

Comment:
Could you be please more specific about the import Errors?

I got:

```python
Traceback (most recent call last):
  File "/usr/bin/snactor", line 9, in <module>
    load_entry_point('leapp==0.9.0', 'console_scripts', 'snactor')()
  File "/usr/lib/python2.7/site-packages/leapp/snactor/__init__.py", line 77, in main
    load_commands()
  File "/usr/lib/python2.7/site-packages/leapp/snactor/__init__.py", line 19, in load_commands
    _load_commands_from(commands.__file__)
  File "/usr/lib/python2.7/site-packages/leapp/snactor/__init__.py", line 33, in _load_commands_from
    mod = importer.find_module(name).load_module(name)
  File "/usr/lib64/python2.7/pkgutil.py", line 246, in load_module
    mod = imp.load_module(fullname, self.file, self.filename, self.etc)
  File "/usr/lib/python2.7/site-packages/leapp/snactor/commands/discover.py", line 8, in <module>
    from leapp.repository.scan import find_and_scan_repositories
  File "/usr/lib/python2.7/site-packages/leapp/repository/__init__.py", line 9, in <module>
    from leapp.repository.actor_definition import ActorDefinition
  File "/usr/lib/python2.7/site-packages/leapp/repository/actor_definition.py", line 10, in <module>
    from leapp.actors import get_actor_metadata, get_actors
  File "/usr/lib/python2.7/site-packages/leapp/actors/__init__.py", line 16, in <module>
    from leapp.workflows.api import WorkflowAPI
  File "/usr/lib/python2.7/site-packages/leapp/workflows/__init__.py", line 16, in <module>
    from leapp.utils.output import display_status_current_phase, display_status_current_actor
  File "/usr/lib/python2.7/site-packages/leapp/utils/output.py", line 8, in <module>
    from leapp.libraries.stdlib.config import is_verbose
  File "/usr/lib/python2.7/site-packages/leapp/libraries/stdlib/__init__.py", line 14, in <module>
    from leapp.libraries.stdlib import api
  File "/usr/lib/python2.7/site-packages/leapp/libraries/stdlib/api.py", line 9, in <module>
    from leapp.actors import Actor
ImportError: cannot import name Actor
```
when running "snactor discover"...

Reply:
@vinzenz any update on this?

Reply:
As discussed offline, this was most probably related to the old build. The new one was stuck.